modules/virt-creating-data-volumes-using-pvc-api.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-pvc-api_{context}"]
+
+= Creating data volumes using the PVC API
+
+When you create a data volume using the *pvc* API, the CDI creates the data volume based on your explicit:
+
+* `accessModes` (`ReadWriteOnce`, `ReadWrtieMany`, `ReadOnlyMany`)
+* `volumeMode` (`Filesystem` or `Block`)
+* `capacity` of `storage` (`5Gi`, for example)
+
+When you explicitly allocate a data volume by using the *pvc* API, you must consider filesystem overhead. Filesystem overhead is the amount of space required by the CDI to manage the file. Failing to account for filesystem overhead in your storage capacity request may result in an underlying PVC that is not large enough to accommodate your virtual machine disk.
```

Comment:
Filesystem overhead is the amount of space required by the filesystem to maintain its own metadata.

Reply:
The amount of space required for filesystem metadata is filesystem dependent.

Reply:
Filesystem overhead is only relevant for filesystem mode PVCs.  There is no filesystem overhead for block mode pvcs

Reply:
Changing to:

_When you explicitly allocate a data volume by using the *pvc* API and you are not using `volumeMode` `Block`, you must consider filesystem overhead. 

Filesystem overhead is the amount of space required by the filesystem to maintain its metadata. The amount of space required for filesystem metadata is filesystem dependent. Failing to account for filesystem overhead in your storage capacity request may result in an underlying PVC that is not large enough to accommodate your virtual machine disk._

Reply:
I restructured this module a bit based on this information and how it pertains to the PVC API. I think it reads better as i relegated much of this filesystem overhead info to a CAUTION. Let me know what you think.